src/test/jolie/test.ol

```diff
@@ -1,2 +1,36 @@
-/** TODO: Test for the db-connector here */
+include "db_connector_iface.iol"
+include "ini_utils.iol"
+include "console.iol"
+include "string_utils.iol"
 
+outputPort DBConnector {
+Interfaces: DBConnectorIface
+}
+
+/*
```

Comment:
Tests should not require any configuration to be run. They should just run and test something.